app/forms/candidate_interface/degree_wizard.rb

```diff
@@ -461,6 +420,75 @@ def specified_grades?
 
   private
 
+    def reviewing_or_country_not_changed_steps(step)
+      case step
+      when :degree_level
+        degree_has_type? ? :type : :review
+      when :completed
+        :award_year
+      when :award_year
+        completed? && international? ? :enic : :review
+      when :enic
+        if international?
+          enic_reason == HAS_STATEMENT ? :enic_reference : :review
+        else
+          :review
+        end
```

Comment:
```suggestion
        if international? || enic_reason == HAS_STATEMENT
          :enic_reference
        else
          :review
        end
```

Reply:
I've edited this to `if international? && enic_reason == HAS_STATEMENT` instead of `||`